docs/migration/5_0.md

```diff
@@ -407,8 +407,6 @@ The following keys have been added to `configurator-common.ts`:
 
 ### AuthHttpHeaderService
 
-- The `refreshInProgress` property was removed. Use `refreshInProgress$` Observable instead.
-- The `handleExpiredToken` method was removed. Use `getValidToken` instead.
 - In `handleExpiredAccessToken` method third parameter `initialToken` is now required.
```

Comment:
Reminder:

Ask Patrick about optional params. I recall these were supported before, but currently it does not handle the case where it was optional and now required